spec/core/numeric/denominator_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../../spec_helper'
+require_relative 'fixtures/classes'
+
+describe "Numeric#denominator" do
+  before :each do
+    @obj = NumericSpecs::Subclass.new
+  end
+
+  it "calls self#to_r then returns the #denominator" do
+    result = mock("Numeric#to_r result")
+    result.should_receive(:denominator).and_return(12)
+    @obj.should_receive(:to_r).and_return(result)
+
+    @obj.denominator.should == 12
+  end
+end
```

Comment:
If you do indeed plan to PR these into ruby/spec upstream, then this is fine. However, our usual technique is to add our tests in the `test/natalie` directory, e.g. `test/natalie/float_test.rb` or similar.

Reply:
I was planning to, but then I just noticed they already exist 🤦 

https://github.com/ruby/spec/blob/master/core/numeric/denominator_spec.rb

I will update PR to add them

Reply:
Ahh yeah, we just copy over files as-needed. Thanks!